### PR TITLE
SW-2105 Pre-commit hook to format docstring

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

File: octoprint_mrbeam/printing/acc_watch_dog.py [hunk 1]
```diff
@@ -174,10 +174,11 @@ def _get_cpu_temp(self):
             self._logger.exception("Excpetion while reading cpu temperature: ")
 
     def _get_cpu_throttle_warnings(self):
-        """
-        See this https://harlemsquirrel.github.io/shell/2019/01/05/monitoring-raspberry-pi-power-and-thermal-issues.html
-        0b1010000000000000000
-          1110000000000000010
+        """See this
+        https://harlemsquirrel.github.io/shell/2019/01/05/monitoring-raspberry-
+        pi-power-and-thermal-issues.html 0b1010000000000000000
```
Comment (JFSolution) [inline octoprint_mrbeam/printing/acc_watch_dog.py]: This destroys the format of the comment

File: octoprint_mrbeam/analytics/cpu.py [hunk 2]
```diff
@@ -109,10 +109,11 @@ def record_cpu_data(self):
...
-        0b1010000000000000000
-          1110000000000000010
+        """See this
+        https://harlemsquirrel.github.io/shell/2019/01/05/monitoring-raspberry-
+        pi-power-and-thermal-issues.html 0b1010000000000000000
+        1110000000000000010.
+
           |||             |||_ under-voltage
           |||             ||_ currently throttled
           |||             |_ arm frequency capped
```
Comment (JFSolution) [inline octoprint_mrbeam/analytics/cpu.py]: here the same

File: octoprint_mrbeam/analytics/cpu.py [hunk 3]
```diff
@@ -109,10 +109,11 @@ def record_cpu_data(self):
...
-        See this https://harlemsquirrel.github.io/shell/2019/01/05/monitoring-raspberry-pi-power-and-thermal-issues.html
-        0b1010000000000000000
-          1110000000000000010
+        """See this
+        https://harlemsquirrel.github.io/shell/2019/01/05/monitoring-raspberry-
+        pi-power-and-thermal-issues.html 0b1010000000000000000
+        1110000000000000010.
+
           |||             |||_ under-voltage
           |||             ||_ currently throttled
```
Comment (JFSolution) [inline octoprint_mrbeam/analytics/cpu.py]:
```suggestion
        """See this
        https://harlemsquirrel.github.io/shell/2019/01/05/monitoring-raspberry-
       pi-power-and-thermal-issues.html
        0b1010000000000000000
          1110000000000000010
          |||             |||_ under-voltage
          |||             ||_ currently throttled
```

**Review by JFSolution — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/printing/acc_watch_dog.py (hunk 1 above)
Comment (irlaec) [reply octoprint_mrbeam/printing/acc_watch_dog.py]: Fixed

File: octoprint_mrbeam/analytics/cpu.py (hunk 2 above)
Comment (irlaec) [reply octoprint_mrbeam/analytics/cpu.py]: Fixed

**Review by JFSolution — APPROVED** [review]

**Pull request merged**